app/models/census/api/census_api.rb

```diff
@@ -18,7 +18,7 @@ def self.send_request
         response = yield
 
         http_response_code = response.code.to_i
-        return { http_response_code: http_response_code } if http_response_code == 500
+        return { http_response_code: http_response_code } if [500, 204].include?(http_response_code)
```

Comment:
mm, i think that 204 is more like 200 than 500

Reply:
Yeah, the thing is that `204` comes without any response body, so if we later try to parse it, we crash. That's why I'm returning early for now.

Reply:
ah, ok :+1: